frameworks/digital-outcomes-and-specialists/questions/briefs/essentialRequirementsOutcomes.yml

```diff
@@ -1,17 +1,18 @@
-name: Essential skills and experience
+name: 'Technical competence: essential skills and experience'
```

Comment:
And the tech competence thing needs to be as clear as possible from the summary and opportunity views as it does the question page itself.